spec/models/manageiq/providers/nuage/network_manager_spec.rb

```diff
@@ -73,6 +73,22 @@
         expect { @ems.verify_credentials(:amqp) }.to raise_error(StandardError)
       end
     end
+
+    context 'credentials encoding' do
+      before do
+        creds = {}
+        creds[:amqp] = {:userid => "amqp_user@!$&", :password => "amqp_password@!$&"}
+        @ems.endpoints << Endpoint.create(:role => 'amqp', :hostname => 'amqp_hostname', :port => '5672')
+        @ems.update_authentication(creds, :save => false)
+      end
+
+      it 'encodes username and password' do
+        # @ems.send(:amqp_urls)
```

Comment:
@gasper-vrhovsek - Can you delete this commented out line if it isn't needed.

Reply:
@bronaghs I apologize for the late fix, the commented line is now removed.